Header header
uint16 object_id
float32 radius
float32 x_local
float32 y_local
float32 x_world
float32 y_world
#float32 vx_world
#float32 vy_world
#float32 abusolute_velocity
#odometry doesn't offer velocity info of car, abusolute velocity of the object can't be calculated
float32 relatice_velocity
